go/libkb/util_windows.go

```diff
@@ -96,3 +108,79 @@ func SafeWriteToFile(g SafeWriteLogger, t SafeWriter, mode os.FileMode) error {
 	}
 	return err
 }
+
+// helper for RemoteSettingsRepairman
+func moveNonChromiumFiles(g *GlobalContext, oldHome string, currentHome string) error {
+	g.Log.Info("RemoteSettingsRepairman moving from %s to %s", oldHome, currentHome)
+	files, _ := filepath.Glob(filepath.Join(oldHome, "*"))
+	for _, oldPathName := range files {
+		_, name := filepath.Split(oldPathName)
+		// Chromium seems stubborn about these - TBD
+		switch name {
+		case "GPUCache":
+			continue
+		case "lockfile":
+			continue
+		case "app-state.json":
+			continue
+		case "Cache":
+			continue
+		case "Cookies":
+			continue
+		case "Cookies-journal":
+			continue
+		case "Local Storage":
+			continue
+		}
+		// explicitly skip logs
+		if strings.HasSuffix(name, ".log") {
+			continue
+		}
+		newPathName := filepath.Join(currentHome, name)
+		var err error
+		g.Log.Info("   moving %s", name)
+
+		for i := 0; i < 5; i++ {
+			if err != nil {
+				time.Sleep(100 * time.Millisecond)
+			}
+			err = os.Rename(oldPathName, newPathName)
+			if err == nil {
+				break
+			}
+		}
+		if err != nil {
+			g.Log.Error("RemoteSettingsRepairman error moving %s to %s - %s", oldPathName, newPathName, err)
+			return err
```

Comment:
That's the way I had it but Max wanted it to bail out - see his review comment of Dec 7, above. I think it would have been much better to keep going.